src/build.d

```diff
@@ -281,10 +283,11 @@ alias versionFile = memoize!(function()
     Dependency dep;
     with (dep)
     {
+        name = "versionFile";
```

Comment:
Not every target needs a name.  Without the name, the help just shows the file that this target generates which in this case I think i s better.